src/background_process.c

```diff
@@ -164,6 +164,16 @@ void *background_process(void *ptr)
 		    log_to_disk(true);
 		    break;
 
+		case QTCRENTRY:
+
+		    store_recv_qtc(lan_message+2);
+		    break;
+
+		case QTCSENTRY:
+
+		    store_sent_qtc(lan_message+2);
+		    break;
+		    
```

Comment:
You use some functions here and let the compilerr guess the prototype. That will sometimes go wrong.
So background_process.c needs to #iinclude "store_recv_qtc.h" and #include "store_sent_qtc.h" file to know the prototypes of the two functions here.